src/module/dy.js

```diff
@@ -118,9 +127,11 @@ module.exports = ({ call, getService, logger }) => ({
         index = null,
         limit = 20,
         consistent = true,
+        sortKeyConstraint = null,
```

Comment:
might rename to `sortKeyConditions`. Thoughts?

Reply:
use `conditions`, then validate with joi schema that only single condition against sort key, then remap to be of format the library expects